specification/maintenance/resource-manager/Microsoft.Maintenance/preview/2022-11-01-preview/Maintenance.json

```diff
@@ -0,0 +1,2564 @@
+        "timeZone": {
+          "description": "Name of the timezone. List of timezones can be obtained by executing [System.TimeZoneInfo]::GetSystemTimeZones() in PowerShell. Example: Pacific Standard Time, UTC, W. Europe Standard Time, Korea Standard Time, Cen. Australia Standard Time.",
+          "type": "string"
+        },
+        "recurEvery": {
+          "description": "Rate at which a Maintenance window is expected to recur. The rate can be expressed as daily, weekly, or monthly schedules. Daily schedule are formatted as recurEvery: [Frequency as integer]['Day(s)']. If no frequency is provided, the default frequency is 1. Daily schedule examples are recurEvery: Day, recurEvery: 3Days.  Weekly schedule are formatted as recurEvery: [Frequency as integer]['Week(s)'] [Optional comma separated list of weekdays Monday-Sunday]. Weekly schedule examples are recurEvery: 3Weeks, recurEvery: Week Saturday,Sunday. Monthly schedules are formatted as [Frequency as integer]['Month(s)'] [Comma separated list of month days] or [Frequency as integer]['Month(s)'] [Week of Month (First, Second, Third, Fourth, Last)] [Weekday Monday-Sunday] [Optional Offset(No. of days)]. Offset value must be between -6 to 6 inclusive. Monthly schedule examples are recurEvery: Month, recurEvery: 2Months, recurEvery: Month day23,day24, recurEvery: Month Last Sunday, recurEvery: Month Fourth Monday, recurEvery: Month Last Sunday Offset-3, recurEvery: Month Third Sunday Offset6.",
+          "type": "string"
+        }
+      }
+    },
+    "MaintenanceOverrideProperties": {
+      "description": "Definition of a MaintenanceOverrideProperties",
+      "type": "object",
+      "properties": {
+        "startDateTime": {
+          "description": "Effective start date of the maintenance override window in YYYY-MM-DD hh:mm format. The start date can be set to either the current date or future date. The window will be created in the time zone provided and adjusted to daylight savings according to that time zone.",
+          "type": "string"
+        },
+        "endDateTime": {
+          "description": "Effective end date of the maintenance override window in YYYY-MM-DD hh:mm format. The window will be created in the time zone provided and adjusted to daylight savings according to that time zone. Expiration date must be set to a future date. If not provided, it will be set to the maximum datetime 9999-12-31 23:59:59.",
+          "type": "string"
+        },
+        "timeZone": {
+          "description": "Name of the timezone. List of timezones can be obtained by executing [System.TimeZoneInfo]::GetSystemTimeZones() in PowerShell. Example: Pacific Standard Time, UTC, W. Europe Standard Time, Korea Standard Time, Cen. Australia Standard Time.",
+          "type": "string"
+        },
+        "overrideProperties": {
+          "description": "Gets or sets overrideProperties of the maintenanceConfiguration",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
>"additionalProperties": { [](http://example.com/codeflow?start=10&length=25)

what is this for? additionalProperties are an anti-pattern that are not very useful for customer facing contracts unless it represents a user-defined properties.

Reply:
User can specify additional information that could relay information as part of the configuration.